src/components/TxBuilderResult.js

```diff
@@ -109,7 +109,7 @@ class TxBuilderResult extends React.Component {
     ) : null;
 
     const txBuilderHasErrors =
-      transactionBuild && transactionBuild.errors.length > 0;
+      transactionBuild && !transactionBuild.errors.length;
```

Comment:
This logic no longer matches the variable name, is that intentional? If so, the name should be updated too.